webapp/package.json

```diff
@@ -13,7 +13,7 @@
     "os": false
   },
   "engines": {
-    "node": ">=8.11.0",
+    "node": ">=12.14.0",
```

Comment:
Is 12+ required by angular now?

Reply:
Yes, we talked about this at a previous product team call and we agreed that we could go forward with this if we could build the app with node 12 and test it with our three test runners using node 8, 10 and 12  
This is what is currently being done in the CI and it works nicely

Are we still okay with this or do we want to hold off merging the upgrade until we can increase our requirements with CHT 4.x?